### ISSUE TITLE
Token classification for a non-textual data

### ISSUE THREAD

**Issue by vitalyshalumov** [huggingface/transformers#21848]
### System Info

- `transformers` version: 4.24.0
- Platform: Linux-5.13.0-37-generic-x86_64-with-glibc2.29
- Python version: 3.8.10
- Huggingface_hub version: 0.10.1
- PyTorch version (GPU?): 1.10.2+cu113 (True)

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

I'm looking for an implementation of an architecture that performs token classification, but the input is not an integer that represents the vocabulary but a vector of numbers.

Basically, each token in the input is represented by a vector. Each token is already an embeddings vector.

How can this be achieved?

Best,
Vitaly

### Expected behavior

Input vector of size 768 for each token. A sequence of such tokens of up to 512.

Maybe it is as simple as removing the layer
 (word_embeddings): Embedding(50265, 768, padding_idx=1)?
In any case a link to the solution would be most helpful.

Best,
Vitaly

**Comment by sgugger** [comment]
You should probably ask this open question on the [forums](https://discuss.huggingface.co/).
